src/v/cloud_storage/tests/topic_manifest_test.cc

```diff
@@ -69,6 +69,24 @@ static constexpr std::string_view full_topic_manifest_json = R"json({
     "retention_duration": 36000000000
 })json";
 
+static constexpr std::string_view topic_manifest_json_with_prev_revision
+  = R"json({
+    "version": 1,
```

Comment:
Is there any need to bump this version? I assume new code treats the new `prev_revision_id` as an optional; it will not fail it is is missing. What will old code do if it reads a manifest with this new field?